crossfire/clients/occurrences.py

```diff
@@ -14,12 +16,13 @@
 except ImportError:
     pass
 
-from crossfire.errors import CrossfireError, RetryAfterError
+from crossfire.errors import CrossfireError, DateError, RetryAfterError
 from crossfire.logger import Logger
 
 logger = Logger(__name__)
 
 TYPE_OCCURRENCES = {"all", "withVictim", "withoutVictim"}
+NUMERIC_VALUES = "[^0-9]"
```

Comment:
O nome diz _valores numéricos_, mas o significado dessa expressão regular é extamente o oposto: tudo que não for valor numérico. Que tal renomear?

Ainda:
* `\d` é uma forma mais sucinta de escrever a mesma coisa
* se vai guardar uma expressão regular em uma contante, compile (`re.compile`) para ter uma performance melhor : )

Reply:
> O nome diz _valores numéricos_, mas o significado dessa expressão regular é extamente o oposto: tudo que não for valor numérico. Que tal renomear?

Perfeito. Falta de conhecimento meu. 

> * `\d` é uma forma mais sucinta de escrever a mesma coisa
> * se vai guardar uma expressão regular em uma contante, compile (`re.compile`) para ter uma performance melhor : )

Ok. Vou ver isso com calma.